### PR TITLE
KEYCLOAK-12410 Deploying themes using Keycloak Operator

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.redhat.com/browse/KEYCLOAK-12410

This Pull Request adds a small clarification how to deploy themes with Keycloak Operator.

### CODE REVIEW

File: topics/templates/document-attributes-product.adoc [hunk 1]
```diff
@@ -51,6 +51,7 @@
 :apidocs_link: {project_doc_base_url}/api_documentation/index
 :developerguide_name: Server Developer Guide
 :developerguide_link: {project_doc_base_url}/server_developer_guide/
+:developerguide_deploying_themes: {developerguide_link}#deploying-themes
```
Comment (ASzc) [inline topics/templates/document-attributes-product.adoc]: The anchor doesn't work, should be an underscore, apparently:

https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html-single/server_developer_guide/index#deploying-themes

vs 

https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html-single/server_developer_guide/index#deploying_themes

**Review by ASzc — CHANGES_REQUESTED** [review]
Other than the broken product anchor, this LGTM.

**Comment by abstractj** [comment]
@slaskawi could you please check Alex's comments?

File: topics/templates/document-attributes-product.adoc (hunk 1 above)
Comment (slaskawi) [reply topics/templates/document-attributes-product.adoc:54]: This is interesting as the underscore doesn't work for the upstream: https://www.keycloak.org/docs/latest/server_development/index.html#deploying-themes

Comment (slaskawi) [reply topics/templates/document-attributes-product.adoc]: You're absolutely right, @ASzc! Fixing.

Comment (slaskawi) [reply topics/templates/document-attributes-product.adoc]: Done.

**Comment by slaskawi** [comment]
Ready to reviewed @ASzc @abstractj

**Comment by slaskawi** [comment]
@ASzc May I ask for a review?

**Review by ASzc — DISMISSED** [review]
Thanks for fixing that anchor, everything else looks ok still.

@abstractj LGTM

**Comment by abstractj** [comment]
@ASzc @slaskawi maybe it's unrelated with the changes in this PR, but I can see the following error at the logs:

```
Results :

Failed tests:   checkVariables(org.keycloak.documentation.test.GettingStartedTest): Variables not found
  checkExternalLinks(org.keycloak.documentation.test.SecuringAppsTest): Invalid links
```

**Comment by slaskawi** [comment]
@abstractj It seems it was unrelated. I rebased my PR and locally all the tests passed.

**Pull request merged**